.circleci/config.yml

```
@@ -1170,3 +1170,15 @@ workflows:
     when: << pipeline.parameters.run_workflow_builder >>
     jobs:
       - docker-build-builder-image
+
+  build-accessibility-testing-weekly:
+    triggers:
+      - schedule:
+          # Every Saturday, at Noon.
+          cron: "0 12 * * 6"
+          filters:
+            branches:
+              only:
+                - main
```

[boulder] I'm not sure if this is correct. The testing site only deploys from the branch `accessibility-testing-repo`.

I think it should be this, but you should verify.

```suggestion
          filters:
            branches:
              only:
                - accessibility-testing-repo
```

---

Fixed.